src/main/java/com/softserve/teamproject/config/WebSecurityConfig.java

```diff
@@ -45,16 +45,18 @@ protected void configure(HttpSecurity http) throws Exception {
         //@formatter:on
   }
 
+
   @Override
   protected void configure(AuthenticationManagerBuilder auth)
       throws Exception {
     auth.authenticationProvider(authenticationProvider());
   }
 
+
   @Bean
   public DaoAuthenticationProvider authenticationProvider() {
     DaoAuthenticationProvider authProvider
-        = new DaoAuthenticationProvider();
+      = new DaoAuthenticationProvider();
```

Comment:
Can you please exclude changes to that file from your branch? They don't look to be necessary